README.md

```diff
@@ -21,7 +21,7 @@ To deploy the machine-api-operator in a vanilla kubernetes environment, one need
     ```
 - Run:
     ```sh
-    ./bin/machine-api-operator --kubeconfig ${HOME}/.kube/config  --config tests/e2e/manifests/mao-config.yaml --manifest-dir manifests
+    ./bin/machine-api-operator start --kubeconfig ${HOME}/.kube/config  --images-json examples/machine-api-operator-config-aws.yaml
```

Comment:
--images-json is coming but not yet implements and you definitely need --config. The command above is correct

Reply:
Ok, will leave this change out until after #73 is finalized